### PR TITLE
[1.18] Fix Fluids Transformation

### LINKED ISSUE #358: [BUG] Barrel making sea water instead of witch water
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I'm playing All The Mods 7 To The Sky. I made a 9x9 patch of mycelium, and placed a bunch of barrels on it. It started to rain, the barrels filled up, and most of them transformed to witch water. A few however, transformed to sea water.

Clicking the witch water with sand gave me soul sand, the barrels start to fill back up, and now more barrels transform to sea water. There's definitely NOT sand underneath the barrel...

After 5 stacks of sand 22 out of 54 barrels contain sea water.

I've also tried using @Pipez fluid pipes and an @CookingForBlockheads sink, and the same happens. The first iteration all 54 barrels transformed into witch water, the second iteration (after clicking the barrels with sand to get soul sand) 5 out of 54 transformed into soul sand

### Expected Behavior

I would expect all barrels to transform to witch water, and none of them to transform to sea water

### Steps to Reproduce

Place mycellium blocks
Place barrels on top
Fill barrel with water
Wait until it transforms to witch water
Doubleclick the barrels with sand in your hand
Fill barrels with water
See that some barrels transform into sea water

### Minecraft Version

1.18.2

### Forge Version

40.1.79

### Ex Nihilo: Sequentia Version

20220818-010806

### Additional Context

ATM 7 To The Sky 1.0.8

### Log File

[logs.zip](https://github.com/NovaMachina-Mods/ExNihiloSequentia/files/9644449/logs.zip)

### PR DESCRIPTION
**Pull request by remplerus**
fix #358

### CODE REVIEW

File: src/main/resources/META-INF/accesstransformer.cfg [hunk 1]
```diff
@@ -1 +0,0 @@
-protected net.minecraft.data.recipes.RecipeProvider m_206406_(Lnet/minecraft/tags/TagKey;)Lnet/minecraft/advancements/critereon/InventoryChangeTrigger$TriggerInstance; # has
```
Comment (NovaMachina) [inline src/main/resources/META-INF/accesstransformer.cfg:1]: Why was this removed?

Comment (remplerus) [reply src/main/resources/META-INF/accesstransformer.cfg:1]: this was removed due to forge has added this AT like 6 months ago

**Review by NovaMachina — APPROVED** [review]
